client/src/components/History/Multiple/MultipleViewList.vue

```diff
@@ -61,12 +61,21 @@ export default {
     data() {
         return {
             MultipleViewItem,
-            selectedHistories: [],
+            selectedHistories: localStorage.getItem("selectedHistories")
+                ? JSON.parse(localStorage.getItem("selectedHistories"))
+                : [],
         };
     },
+    watch: {
```

Comment:
No need to watch it here.

Reply:
This might make more sense to persist within the store itself, instead of having an extra layer here?

Reply:
That is a good point, so create a `VueX` store instead, called e.g. `store/historyStore/historyMultipleStore.js` and then register with `VuexPersistence` in `store/index.js`?

Reply:
Ideally in the same hsitoryStore as `recentyViewedHistories` or something slightly more flexible than "histories pinned to multiview" -- then we can use this in other spots for quick access.